protocol/v2/ssv/runner/committee.go

```diff
@@ -414,95 +415,99 @@ func (cr *CommitteeRunner) ProcessPostConsensus(logger *zap.Logger, signedMsg *s
 	}
+	submmitionStart := time.Now()
+	if err := cr.beacon.SubmitAttestations(attestations); err != nil {
+		logger.Error("❌ failed to submit attestation", zap.Error(err))
+		return errors.Wrap(err, "could not submit to Beacon chain reconstructed attestation")
+	}
 
 	if len(attestations) > 0 {
-		submissionStart := time.Now()
-		if err := cr.beacon.SubmitAttestations(attestations); err != nil {
-			logger.Error("❌ failed to submit attestation", zap.Error(err))
-			return errors.Wrap(err, "could not submit to Beacon chain reconstructed attestation")
-		}
-
 		logger.Info("✅ successfully submitted attestations",
-			fields.SubmissionTime(time.Since(submissionStart)),
 			fields.Height(cr.BaseRunner.QBFTController.Height),
-			fields.Round(cr.BaseRunner.State.RunningInstance.State.Round))
-		// Record successful submissions
-		for validator := range attestationsToSubmit {
-			cr.RecordSubmission(spectypes.BNRoleAttester, validator)
-		}
+			fields.Round(cr.BaseRunner.State.RunningInstance.State.Round),
+			fields.Root([32]byte(attestations[0].Data.BeaconBlockRoot[:])),
+			fields.SubmissionTime(time.Since(submmitionStart)),
+			zap.Float64("total_consensus_time", time.Since(cr.started).Seconds()))
+	}
+	// Record successful submissions
+	for validator := range attestationsToSubmit {
+		cr.RecordSubmission(types.BNRoleAttester, validator)
 	}
 
 	// Submit multiple sync committee
-	syncCommitteeMessages := make([]*altair.SyncCommitteeMessage, 0, len(syncCommitteeMessagesToSubmit))
+	syncCommitteeMessages := make([]*altair.SyncCommitteeMessage, 0)
 	for _, syncMsg := range syncCommitteeMessagesToSubmit {
 		syncCommitteeMessages = append(syncCommitteeMessages, syncMsg)
 	}
+	submmitionStart = time.Now()
+	if err := cr.beacon.SubmitSyncMessages(syncCommitteeMessages); err != nil {
+		logger.Error("❌ failed to submit sync committee", zap.Error(err))
+		return errors.Wrap(err, "could not submit to Beacon chain reconstructed signed sync committee")
+	}
 	if len(syncCommitteeMessages) > 0 {
-		submissionStart := time.Now()
-		if err := cr.beacon.SubmitSyncMessages(syncCommitteeMessages); err != nil {
-			logger.Error("❌ failed to submit sync committee", zap.Error(err))
-			return errors.Wrap(err, "could not submit to Beacon chain reconstructed signed sync committee")
-		}
 		logger.Info("✅ successfully submitted sync committee",
-			fields.SubmissionTime(time.Since(submissionStart)),
 			fields.Height(cr.BaseRunner.QBFTController.Height),
-			fields.Round(cr.BaseRunner.State.RunningInstance.State.Round))
-		// Record successful submissions
-		for validator := range syncCommitteeMessagesToSubmit {
-			cr.RecordSubmission(spectypes.BNRoleSyncCommittee, validator)
-		}
+			fields.Round(cr.BaseRunner.State.RunningInstance.State.Round),
+			fields.Root([32]byte(syncCommitteeMessages[0].BeaconBlockRoot[:])),
+			fields.SubmissionTime(time.Since(submmitionStart)),
+			zap.Float64("total_consensus_time", time.Since(cr.started).Seconds()))
```

Comment:
use consistent duration string like you use in other durations, move it to fields.TotalConsensusTime